doc/releases/changelog-dev.md

```diff
@@ -524,64 +505,49 @@ Deprecations cycles are tracked at [doc/developement/deprecations.rst](https://d
-  A new argument `seed` has been added, which defaults to None and can contain an integer with the
-  wanted seed.
+* The `seed_recipes` argument in `qml.classical_shadow` and `qml.shadow_expval` is deprecated. A new argument `seed` has been added, which defaults to None and can contain an integer with the wanted seed.
   [(#3388)](https://github.com/PennyLaneAI/pennylane/pull/3388)
 
-* `make_tape` is deprecated. Please use `qml.tape.make_qscript` instead.
+* `qml.transforms.make_tape` has been deprecated. Please use `qml.tape.make_qscript` instead.
   [(#3478)](https://github.com/PennyLaneAI/pennylane/pull/3478)
 
 <h3>Documentation</h3>
 
-* Adds developer documentation for the queuing module.
+* Developer documentation for the queuing module has been added.
   [(#3268)](https://github.com/PennyLaneAI/pennylane/pull/3268)
 
-* Corrects more mentions for diagonalizing gates for all relevant operations. The docstrings for `compute_eigvals` used
-  to say that the diagonalizing gates implemented $U$, the unitary such that $O = U \Sigma U^{\dagger}$, where $O$ is
-  the original observable and $\Sigma$ a diagonal matrix. However, the diagonalizing gates actually implement
-  $U^{\dagger}$, since $\langle \psi | O | \psi \rangle = \langle \psi | U \Sigma U^{\dagger} | \psi \rangle$, making
-  $U^{\dagger} | \psi \rangle$ the actual state being measured in the $Z$-basis.
+* More mentions of diagonalizing gates for all relevant operations have been corrected. The docstrings for `compute_eigvals` used to say that the diagonalizing gates implemented $U$, the unitary such that $O = U \Sigma U^{\dagger}$, where $O$ is the original observable and $\Sigma$ a diagonal matrix. However, the diagonalizing gates actually implement $U^{\dagger}$, since $\langle \psi | O | \psi \rangle = \langle \psi | U \Sigma U^{\dagger} | \psi \rangle$, making $U^{\dagger} | \psi \rangle$ the actual state being measured in the $Z$-basis.
   [(#3409)](https://github.com/PennyLaneAI/pennylane/pull/3409)
 
 <h3>Bug fixes</h3>
 
-* Fixed a bug where `hamiltonian_expand` didn't preserve the type of the inputted results in its output.
+* Fixed a bug where `qml.transforms.hamiltonian_expand` didn't preserve the type of the input results in its output.
   [(#3339)](https://github.com/PennyLaneAI/pennylane/pull/3339)
 
-* Fixed a bug that made `gradients.param_shift` raise an error when used with unshifted terms only
-  in a custom recipe, and when using any unshifted terms at all under the new return type system.
+* Fixed a bug that made `qml.gradients.param_shift` raise an error when used with unshifted terms only in a custom recipe, and when using any unshifted terms at all under the new return type system.
   [(#3177)](https://github.com/PennyLaneAI/pennylane/pull/3177)
 
 * Original tape `_obs_sharing_wires` attribute is updated during its expansion.
+  TODO: This PR was made in v0.27 ?
```

Comment:
Please remove once addressed

Reply:
I checked the v0.27 changelog, couldn't find any reference to this